packages/pn-pa-webapp/public/robots.txt

```diff
@@ -1,3 +1,7 @@
 # https://www.robotstxt.org/robotstxt.html
 User-agent: *
-Disallow:
+Disallow: /dashboard
+Disallow: /dashboard/
+Disallow: /api-keys
+Disallow: /api-kesy/
+Disallow: /app-status
```

Comment:
The new notification route is missed

Reply:
Is included in `/dashboard/`. Beacuse the route is `/dashboard/nuova-notifica` and `/dashboard/` disallow all paths that starts with `/dashboard`

Reply:
right!